frontend/src/pages/team/Devices/index.vue

```diff
@@ -92,6 +93,7 @@ export default {
     name: 'TeamDevices',
     data () {
         return {
+            loading: false,
```

Comment:
We're missing the code to set `loading=true` when retrieving the list of devices